Doc/howto/descriptor.rst

```diff
@@ -282,7 +282,7 @@ this::
         . . .
         def __get__(self, obj, objtype=None):
             "Simulate func_descr_get() in Objects/funcobject.c"
-            return types.MethodType(self, obj, objtype)
+            return types.MethodType(self, obj)
```

Comment:
A relic of the unbound-method days. The signature should also be updated since `objtype` is of no use.